docs/completeExamples/completeExamples.md

```diff
@@ -0,0 +1,44 @@
+# Complete Examples
```

Comment:
If this is the main md file in this dir then either it should be called README.md or there should be a symlink called README.md pointing to it.  That way, when you navigate around the repo in github it renders the main page in each folder.

Reply:
fixed this in #26